deploy/osd-storage/config.yaml

```
@@ -0,0 +1,9 @@
+deploymentMode: SelectorSyncSet
+selectorSyncSet: 
+  matchExpressions: 
```

We want to only create and not continuously update: https://github.com/openshift/hive/blob/master/pkg/apis/hive/v1/syncset_types.go#L37
```suggestion
  applyBehavior: "CreateOnly"
  matchExpressions: 
```